hudi-cli/src/main/java/org/apache/hudi/cli/commands/CommitsCommand.java

```diff
@@ -145,13 +148,16 @@ private String printCommitsWithMetadata(HoodieDefaultTimeline timeline,
             .addTableHeaderField("Total Rollback Blocks").addTableHeaderField("Total Log Records")
             .addTableHeaderField("Total Updated Records Compacted").addTableHeaderField("Total Write Bytes");
 
-    return HoodiePrintHelper.print(header, new HashMap<>(), sortByField, descending, limit, headerOnly, rows);
+    return HoodiePrintHelper.print(header, new HashMap<>(), sortByField, descending,
+            limit, headerOnly, rows, tempTableName);
   }
 
   @CliCommand(value = "commits show", help = "Show the commits")
   public String showCommits(
       @CliOption(key = {"includeExtraMetadata"}, help = "Include extra metadata",
           unspecifiedDefaultValue = "false") final boolean includeExtraMetadata,
+      @CliOption(key = {"exportToTableName"}, mandatory = false, help = "hive table name to export",
```

Comment:
This is not a real hive table. So maybe reword this for clarity. Also, export means something persistent while this is a temporary view. 

Name of in-memory view to cache results.

Reply:
Changed names to view instead of table. Initial idea was to create actual tables with all the metadata and register them. so thats where name came from. But, i think exporting to external table requires lot more work with the way CLI is setup. Just doing views in CLI for now. I can work on writing a tool outside CLI to export metadata to another table.